### ISSUE TITLE
`1.36.0` examines more than the latest commit message, ending up setting `#none` as the DEFAULT_BUMP for any subsequent commit

### ISSUE THREAD

**Issue by dimisjim** [anothrNick/github-tag-action#124]
Hey there,

Thanks for the awesome action.

I wanted to try out the `#none` feature that was recently released with 1.36.0.
It currently has this issue that, if you have set the fetch-depth of the previous checkout action to `0`, it will try to examine more than the latest commit.

So for example, if one uses the `none` directive and has the default bump to something else, first time it is was it works fine, but the second time, if you don't specify any other directive, it should work, but instead, the tag action checks the previous commit as well, thus ending up skipping the bump.

This is how this looks like in the actions verbose output:

```
*** CONFIGURATION ***
	DEFAULT_BUMP: patch
	WITH_V: true
	RELEASE_BRANCHES: master,main
	CUSTOM_TAG: 
	SOURCE: .
	DRY_RUN: false
	INITIAL_VERSION: 0.0.0
	TAG_CONTEXT: repo
	PRERELEASE_SUFFIX: beta
	VERBOSE: true
Is master a match for master
Is main a match for master
pre_release = false
DOP-1174 fix readme DOP-1174 #none
Default bump was set to none. Skipping...
```

so basically my first commit was `DOP-1174 #none` which worked as expected and the next one was `DOP-1174 fix readme` but as shown above it didn't work as expected, and ended up skipping the tag bump.

I managed to make this work as expected by switching to fetch-depth to `1`, but I am not sure whether this is a robust solution.

**Comment by sammcj** [comment]
Hello - is this still an issue with v1.45.0 and required?

**Comment by sbe-arg** [comment]
I reckon this has been resolved closing, please reopen if necesary

**Comment by dimisjim** [comment]
>I reckon this has been resolved closing, please reopen if necesary

Is this reckoning derived from a proven test, or just pure speculation?
